src/content/docs/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx

```diff
@@ -135,74 +135,96 @@ with [Amazon CloudWatch Metric Streams](/install/aws-cloudwatch/). This is not
-generates ElastiCache entities in New Relic. These entities will have the
-following entity tags:
-
-* `aws.elasticache.configurationEndpointAddress` or `aws.elasticache.endpointAddress`
-* `aws.elasticache.configurationEndpointPort` or `aws.elasticache.endpointPort`
-
-If your services use OpenTelemetry instrumentation for Memcached it must emit
-the following attributes for a relationship to be generated:
-
-* `db.system`: This attribute must have the value `memcached`.
-* `net.peer.name`: This attribute must match the corresponding endpoint tag of
-  the ElastiCache entity.
-* `net.peer.port`: This attribute must match the corresponding port tag of the
-  ElastiCache entity.
-
-#### Amazon MQ (ActiveMQ)
-
-The Amazon CloudWatch Metric Streams integration for
-[Amazon MQ](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-mq-integration)
-generates Amazon MQ entities in New Relic. These entities will have the
-following entity tag:
 
-* `aws.mq.endpoint`
-
-If your services use OpenTelemetry instrumentation for ActiveMQ it must
-the following attributes for a relationship to be generated:
-
-* `messaging.system`: This attribute must have the value `activemq`.
-* `server.address`: This attribute must match the corresponding endpoint tag
-  of the Amazon MQ entity.
-* `span.kind`: This relationship is generated from span data where `span.kind`
-  is `producer` or `consumer`.
-
-#### Amazon Managed Kafka (MSK)
-
-The Amazon CloudWatch Metric Streams integration for
-[MSK](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-managed-kafka-msk-integration)
-generates MSK entities in New Relic. These entities will have the
-following entity tag:
-
-* `aws.clusterName` or `aws.kafka.ClusterName`
-* `aws.awsRegion` or `aws.region`
-* `aws.topic` or `aws.kafka.Topic`
-
-If your services use OpenTelemetry instrumentation for MSK it must have
-the following attributes for a relationship to be generated:
+  The relationships not supported by instrumentation require you to manually
+  emit telemetry to signal to New Relic that a relationship should be created.
+  To do so, you must emit a log (preferred) or a span with the attribute
+  `newrelic.aws_metric_streams.arn` containing the value of the
+  [ARN](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html)
+  for the resource you are calling. Note that relationships created have a TTL
+  of 75 minutes, so this telemetry must continue to be sent periodically.
+</Callout>
 
-* `messaging.destination.name`: The MSK topic to which the service is either producing messages or consuming messages.
-* `server.address`: This attribute must match the corresponding endpoint tag
-  of the MSK entity.
-* `span.kind`: This relationship is generated from span data where `span.kind`
-  is `producer` or `consumer`.
+<table>
+  <thead>
+    <tr>
+      <th>
+        Supported by instrumentation
```

Comment:
For this column, given that not all instrumentations in all languages will contain the data required for relationships, do you think its worth it to discuss what the specific requirements are of instrumentation?

Reply:
I was trying to avoid that at least to begin with. In general, I think the rules are decent and will cover most of the instrumentation that exists out there. That said, if we discover customers are struggling then we can figure out how best to elaborate.